rules/windows/registry/registry_set/registry_set_office_outlook_enable_unsafe_client_mail_rules.yml

```
@@ -0,0 +1,27 @@
+title: Outlook Security EnableUnsafeClientMailRules Value Was Enabled
```

```suggestion
title: Outlook Security EnableUnsafeClientMailRules Setting Enabled
```
`Was Enabled` gives me the impression of something long in the past